app/screens/create_or_edit_channel/create_or_edit_channel.tsx

```diff
@@ -0,0 +1,254 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState, useEffect, useReducer, useCallback, useMemo} from 'react';
+import {useIntl} from 'react-intl';
+import {
+    Keyboard,
+} from 'react-native';
+import {Navigation} from 'react-native-navigation';
+
+import {patchChannel as handlePatchChannel, createChannel, switchToChannelById} from '@actions/remote/channel';
+import CompassIcon from '@components/compass_icon';
+import {General} from '@constants';
+import {useServerUrl} from '@context/server';
+import {useTheme} from '@context/theme';
+import ChannelInfoForm from '@screens/create_or_edit_channel/channel_info_form';
```

Comment:
```suggestion
import ChannelInfoForm from './channel_info_form';
```

and then you would need to sort the imports.